go/kbfs/libkbfs/conflict_resolver.go

```diff
@@ -109,6 +109,7 @@ type ConflictResolver struct {
 	lockNextTime  bool
 	canceledCount int
 
+	failModeLock    sync.RWMutex
 	failModeForTest failModeForTest
```

Comment:
maybe rename this since it's not for tests only per se anymore?  No strong feelings though.

Reply:
I mean, it's still only used in testing.  Would "ForTesting" be better?

Reply:
Sure.